entrypoint.sh

```diff
@@ -2,4 +2,10 @@
 
 source /opt/ros/galactic/setup.bash
 
-micrortps_agent -t UDP -n "$DRONE_DEVICE_ID"
+if [ "${FLIGHT_CONTROLLER_DIRECT_ETH}" != "" ]; then
+  # Direct ethernet connection to FC
+  micrortps_agent -t UDP -b 10000000 -i 192.168.200.101 -n "$DRONE_DEVICE_ID"
+else
+  # FC connection via protocol_splitter/UART
+  micrortps_agent -t UDP -b 2000000 -n "$DRONE_DEVICE_ID"
```

Comment:
In the `fogsw_systemd/micrortps_agent.service`, there are more parameters passed through. Is the whitelist argument (-a) required? Also, the namespace is not passed to the agent.

Here is the service command in fogsw_systemd.  

`micrortps_agent -b 2000000 -t UDP -r 2020 -s 2019 -n $DRONE_DEVICE_ID -a 127.0.0.1`

Reply:
The port parameters "-r" and "-s" are not needed as the default values has been already correct ones.

The whitelist param(s) is a bit question mark how we should use it in VM env. Need to discuss with that.

Reply:
Great observation @mehmetkillioglu!

To complicate things further, fog_hyper actually overrides the systemd unit file (from fogsw_systemd) with its own: https://github.com/tiiuae/fog_hyper/blob/ca7d95472f584b6418aba633e72a82becc2c8fcc/pkg/foghyper/fogubuntu-fixes/micrortps_agent.service#L10

I had commented the change like this: https://github.com/tiiuae/fog_hyper/blob/ca7d95472f584b6418aba633e72a82becc2c8fcc/pkg/foghyper/debugfogubuntu.go#L190

Reply:
I think systemd service file is not an issue there, it works as it is now in fogubuntu. But when we switch micrortps_agent to it's own container and start using the entrypoint script, we should have the whitelist addresses defined so that the px4 topics are visible to relevant nodes, or should we? 
How the inter-vm network works now in ROS2 point of view? Micrortps agent is running in different VM than connection VM and the mesh network is in different subnet than the drone internal "inter-VM" network? I guess then we don't need to define whitelist addresses because the VM where micrortps_agent runs does not have any network_if where topics should not be delivered, right?

Reply:
Or actually, only think that matters is that if the VM where the node is running has any network interfaces that need to be left out from ROS2 communication then the whitelist shall be define and the IP of that/those network IF(s) shall be left out from the list. If the VM does not have any "restricted" interface, then actually nothing needs to be (or can be) done.

Reply:
So, do you agree that because the micrortps_agent is anyway very likely not running in communication VM, we don't need to define the whitelist there (the -a flag)?

Reply:
Sorry, I have made very poor choices with my GitHub notifications (will need to resolve that)

You're correct, it is unlikely that we'll be running this agent in communication VM! Census has recommended that wireless interfaces have dedicated VMs due to security reasons.